ui/tests/helpers/codemirror.js

```diff
@@ -1,18 +1,6 @@
-const invariant = (truthy, error) => {
-  if (!truthy) throw new Error(error);
-};
-
-export function getCodeMirrorInstance(container) {
-  return function (selector) {
-    const cmService = container.lookup('service:code-mirror');
-
-    const element = document.querySelector(selector);
-    invariant(element, `Selector ${selector} matched no elements`);
-
-    const cm = cmService.instanceFor(element.id);
-    invariant(cm, `No registered CodeMirror instance for ${selector}`);
-
-    return cm;
+export function getCodeMirrorInstance() {
+  return function () {
+    return document.querySelector('.CodeMirror').CodeMirror;
```

Comment:
`suggestion`: We can deprecate this helper entirely now this its just selecting a DOM element without tapping into the container registry or throwing an error.

Its used in 3 files. And we should be able to write:

```
const expected = document.querySelector('.CodeMirror').CodeMirror.value;
const actual = JSON.stringify(result, null, 2);

assert.equals(expected, actual);
```